homeassistant/components/generic/manifest.json

```diff
@@ -1,6 +1,6 @@
 {
   "domain": "generic",
   "name": "Generic",
-  "documentation": "https://www.home-assistant.io/integrations/generic",
+  "documentation": "https://www.home-assistant.io/integrations/generic_ip_camera",
```

Comment:
This will not work for a lot of cases in our codebase at this point.

What needs to be happening, is that we adjust our documentation to have consistent paths.

Reply:
OK.  Shall I close this PR for now then?
I noticed this problem while working on a config_flow for this integration: the automatic link on the UI integrations page didn't work.

Reply:
Yeah, lets close this and fix the documentation side of it.